offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -76,6 +98,39 @@ fun ApolloClient.enqueue(
     this.mutate(mutation).enqueue(apolloCallback)
 }
 
+/*
+ This function is called whenever we get a conflict.
+ Here the mutation with the correct data is retried again.
+ @param mutation: Mutation<Operation.Data, Any, Operation.Variables>
+
+ */
+fun retryConflictedMutation(mutation: Mutation<Operation.Data, Any, Operation.Variables>) {
+
+    /*
+      1. Map the json response String to the Conflict pojo class.
+      2. Extract the correct version from the server data.
+      3. Set the client version to the correct one. (server+1)
+      4. Again make a call with the mutation to the server.
+    */
+    var parsedObject = gson.fromJson(conflcitDetected, ConflictPojo::class.java)
```

Comment:
Instead of the `ConflictPojo` class, we'll have to think of a generic solution as I outlined above.
It shouldn't be too hard.

We can also use choose to use normal JSON handling in the library to just extract the `serverState` key and then use 
GSON to map it with a POJO if we don't need other keys from the server response.

Reply:
@Lavanyagaur22 This is the line that errors in CircleCi.
https://circleci.com/gh/aerogear/offix-android/221?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link